### ISSUE TITLE
Check license presence without generating `$error`s in helper file(s)

### ISSUE THREAD

**Issue by aetonsi** [chocolatey/choco#3218]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

Hi, i'd like to suggest a simple micro improvement. 
The following lines in the "Tab Expansion" helper check for the presence of a license file:
https://github.com/chocolatey/choco/blob/06acb28162b6ee3035acafe91561d8969244f400/src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1#L64-L70
But it does so by relying on a `try{}catch`ed `Get-Item -ErrorAction Stop`. This doesn't _print_ any error, but it still populates the automatic variable `$Error` with an unneeded exception.
I am importing `chocolateyProfile.psm1` which in turn imports this file, in my powershell's `$PROFILE` file.
At the end of my profile file i do a simple error checking on `$Error`. Ever since chocolatey v2 came out, i always have this false positive:
```log
> $error

Get-Item: C:\ProgramData\chocolatey\helpers\ChocolateyTabExpansion.ps1:66:20
Line |
  66 |  … censeFile = Get-Item -Path "$env:ChocolateyInstall\license\chocolatey …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\ProgramData\chocolatey\license\chocolatey.license.xml' because it does not exist.
```
This also causes the `$Error` variable to always start off with 1 item in every single PS session.

### Describe The Solution. Why is it needed?

A very simple solution would be to use `Test-Path` instead of `Get-Item`, and then use `get-item` to fetch the file if needed.

### Additional Context

-

### Related Issues

(maybe) #2851

**Comment by corbob** [comment]
Well this is an interesting one... I am seeing this on a VM, but it looks like there should be Pester tests testing this (at least within PowerShell 2... As well as this code: https://github.com/chocolatey/choco/blob/06acb28162b6ee3035acafe91561d8969244f400/src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1#L113-L119

Which should be removing the error (and when I manually run those, the error is removed).

**Comment by aetonsi** [comment]
Well i don't know why the `$error.removeat(0)` doesn't work, the error just spills into the powershell session.
Maybe that `$error` references a _local_ `$error` (like `$local:error`), and so my global `$error` is not affected by `.removeat(0)`? Just speculating..
Anyway, thanks for the fix, will this be integrated in a minor or in a patch version?

**Comment by corbob** [comment]
> Anyway, thanks for the fix, will this be integrated in a minor or in a patch version?

@aetonsi the fix is assigned to the 2.1.0 milestone, so it should come out with that release when it is ready. At this point I can't say for sure when that might be, but I do expect it to be the next release.

In the meantime, a workaround you could consider is moving the test for errors to before the Chocolatey profile loading, and thus circumvent this issue.

**Comment by aetonsi** [comment]
Thank you again, have a nice day.
